### ISSUE TITLE
Bookmark + Remember last position 

### ISSUE THREAD

**Issue by hnadimi** [futo-org/grayjay-android#1141]
### Use Cases

It would be great to be able to bookmark a position in a video.
Also when closing a video somewhere in the middle it would be nice to resume from the last position upon reopening.

**Comment by lmsalustri** [comment]
Does it not do this already?

**Comment by hnadimi** [comment]
You're right. Resume is already there, one just have to click on it when the video restarts.

Bookmark functionality I couldn't find though.

**Comment by jeffreyd00** [comment]
It would be nicer still of there was a setting for it to automatically resume instead of having to click on the overlay that says Resume at x:xx min

**Comment by Zvonimir-FUTO** [comment]
> It would be nicer still of there was a setting for it to automatically resume instead of having to click on the overlay that says Resume at x:xx min

This works when opening videos from history for now.

**Comment by jeffreyd00** [comment]
that's helpful, thank you
